CoastalResilienceNetwork/GeositeFramework
# Implement infographics for plugins

## Overview

Adds a configuration option to plugins for infographics. When enabled, a button is added to the plugin title bar, which, when clicked, will open the provided infographic content in a modal.

Note that this approach is similar to what was implemented for the launchpad in #1020. However, because of differences in configuration (the launchpad is configured via the region.json file, while the plugins are configured directly in the code), the implementations could not be merged.

Connects to #1025

### Demo

![tnc1](https://user-images.githubusercontent.com/1042475/33279504-c7b0f450-d36c-11e7-92db-34ee0d7927a5.gif)

## Testing Instructions

- Open the Identify Point plugin.
- Click the infographic button in the title bar.
- Verify the infographic opens in a modal.
